pkg/client/client.go

```diff
@@ -254,6 +275,8 @@ func (o *OptimizelyClient) GetAllFeatureVariables(featureKey string, userContext
 		return enabled, variableMap, nil
 	}
 
+	errs := new(multierror.Error)
```

Comment:
This is an improvement since before we only propagated the _last_ error. Thanks.